src/charm.py

```diff
@@ -219,6 +228,13 @@ def __init__(self, *args):
                 }
             ],
         )
+        self.ingress = IngressPerAppRequirer(
+            charm=self,
+            port=self.VAULT_PORT,
+            strip_prefix=True,
+            scheme=lambda: "https",
```

Comment:
Can't this be reduced to:
```suggestion
            scheme="https",
```
?

Reply:
We can not do this. Scheme is expected to be a callable. https://github.com/canonical/traefik-k8s-operator/blob/main/lib/charms/traefik_k8s/v2/ingress.py#L549

Reply:
Ok, thanks!